dbus-codegen/src/generate.rs

```diff
@@ -691,6 +696,11 @@ pub fn generate(xmldata: &str, opts: &GenOpts) -> Result<String, Box<dyn error::
                 if curm.is_some() { Err("End of Interface inside method")? };
                 if curintf.is_none() { Err("End of Interface outside interface")? };
                 let intf = curintf.take().unwrap();
+                // If filters are set and no filter matches -> Just print a message and continue parsing
+                if !opts.filter.is_empty() && !opts.filter.contains(&intf.shortname) {
+                    eprintln!("Skip filtered interface '{}'", &intf.shortname);
+                    continue;
```

Comment:
The `eprintln` there might be useful for debugging during development, but for the end user it's probably more of a distraction, so delete it or comment it out.